projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -673,12 +735,38 @@ private boolean exit() {
    private void generateDatamodel() {
       try {
          ObjectMapper mapper = new BatfishObjectMapper();
-
          JsonSchemaGenerator schemaGenNew = new JsonSchemaGenerator(mapper);
          JsonNode schemaNew = schemaGenNew
                .generateJsonSchema(Configuration.class);
          _logger.output(mapper.writeValueAsString(schemaNew));
 
+//         Reflections reflections = new Reflections("org.batfish.datamodel");
```

Comment:
Actually since it has nothing to do with analysis I am happy to merge it as comments. You can uncomment in another pull request.